specification/langRef/technicalContent/bookrestriction.dita

```diff
@@ -25,9 +30,13 @@
       <p id="attr-exception" outputclass="attr-exception">For this element, the <xmlatt>value</xmlatt> attribute specifies any
         restrictions on the use of the material, such as declaring the information confidential or
         for licensed use only.</p>
+      <draft-comment author="tammy">This seems redundant since it's also captured in the Usage
```

Comment:
We need to keep it in the attribute section, which is the authoritative description of what the attributes mean or allow.